velox/docs/configs.rst

```diff
@@ -108,3 +108,16 @@ can be used to prevent a query from using too much io and cpu resources.
     * **Default value:** ``0``
 
 The maximum allowed spill file size. Zero means unlimited.
+
+``min-spill-run-siz``
+^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``integer``
+    * **Default value:** ``0``
+
+The min spill run size limit used to select partitions for spilling. The
```

Comment:
It would be nice to specify exact units (bytes) and spell out words fully (minimum vs. min).